### PR TITLE
3 long

### PR DESCRIPTION
**Pull request by CurtisColwell**
This adds a separate script for running jobs on long nodes.

### CODE REVIEW

File: scripts/submit_gaussian.bash [hunk 1]
```diff
@@ -6,6 +6,11 @@ WORKING_DIR=$(dirname "$SCRIPTS_DIR")
 INPUT_DIR="$WORKING_DIR/input"
 OUTPUT_DIR="$WORKING_DIR/output"
 LOG_DIR="$WORKING_DIR/logs"
+PARTITION=$1
+
+if [ -z "$PARTITION" ]; then
```
Comment (icolwell) [inline scripts/submit_gaussian.bash]: This if statement can be replaced with:
`: ${PARTITION:="short"}`

File: README.md [hunk 2]
```diff
@@ -25,6 +25,17 @@ everything is working properly. Output, checkpoint and formatted checkpoint
...
 
+### Running on different nodes
+
+If your job needs more time, you can run on a different node. The default node is 'short' 
+which has a 24 h time limit. Switching to 'long' changes the time limit to 2 weeks. A full 
+list of nodes can be found [here](https://hpcrcf.atlassian.net/wiki/spaces/TCP/pages/7285967/Partition+List).
+To run on a different node add the name of the node at the end of the submission command. For 
+example to run on a **long** node run the following command:
+```
+bash auto_slurm/scripts/submit_gaussian.bash long
```
Comment (icolwell) [inline README.md:36]: Nice! much cleaner than having 2 copies of the same code laying around. 
I'm curious, why would anyone ever submit on the short node? would it make more sense to default to long?
Then your jobs that happen to run for 25 hours don't get cancelled.
Is there typically much less jobs on short, so you don't have to wait in a queue?

File: README.md [hunk 3]
```diff
@@ -25,6 +25,17 @@ everything is working properly. Output, checkpoint and formatted checkpoint
 files will be generated in the output directory. A log showing the steps
 performed by the script will be available in the log directory.
 
+### Running on different nodes
+
+If your job needs more time, you can run on a different node. The default node is 'short' 
+which has a 24 h time limit. Switching to 'long' changes the time limit to 2 weeks. A full 
+list of nodes can be found [here](https://hpcrcf.atlassian.net/wiki/spaces/TCP/pages/7285967/Partition+List).
```
Comment (icolwell) [inline README.md:32]: This information seems to be specific to UOregon, don't you have a fork of this repo specifically for your lab and UOregon use cases? or would you rather just leave it here?

File: scripts/submit_gaussian.bash [hunk 4]
```diff
@@ -46,7 +51,7 @@ check_for_jobs()
 	echo "Output, checkpoint and formatted checkpoint files will be placed under:"
 	echo "$OUTPUT_DIR/"
 	echo "Use the following command to check on your job queue:"
-	echo '	squeue -u $USER'
+	echo '	squeue -u' $USER
```
Comment (icolwell) [inline scripts/submit_gaussian.bash]: Is this branch up to date with master? This change is already made on the master branch.

File: README.md (hunk 3 above)
Comment (CurtisColwell) [reply README.md:32]: I'd rather just use this as the lab specific one.

File: README.md (hunk 2 above)
Comment (CurtisColwell) [reply README.md:36]: Yep, the last one.

File: scripts/submit_gaussian.bash (hunk 4 above)
Comment (CurtisColwell) [reply scripts/submit_gaussian.bash]: I made the branch before that change.

File: scripts/submit_gaussian.bash (hunk 1 above)
Comment (CurtisColwell) [reply scripts/submit_gaussian.bash]: what does : ${PARTITION:="short"} mean exactly?

Comment (icolwell) [reply scripts/submit_gaussian.bash]: Sets a default basically, see [this question](https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts), actually, we can remove line 9 too and simply put:
`PARTITION="${1:="short"}"`

File: scripts/submit_gaussian.bash (hunk 4 above)
Comment (icolwell) [reply scripts/submit_gaussian.bash]: ya, if you merge master into this branch then this change shouldn't appear.

**Comment by CurtisColwell** [comment]
Ok, I had to make one small change, but the script works now and can submit to any node.

**Review by icolwell — APPROVED** [review]

**Pull request merged**